Velaptor/InternalExtensionMethods.cs

```diff
@@ -1582,4 +1582,22 @@ public static int IndexOf<T>(this Memory<RenderItem<T>> items, Predicate<T> pred
 
         return -1;
     }
+
+    /// <summary>
+    /// Increases the total amount of the given <paramref name="items"/> by the given <paramref name="amount"/>.
+    /// </summary>
+    /// <param name="items">The items to increase in number.</param>
```

Comment:
Grammar. Not sure what the meaning is, so my fix may not make sense. 
```suggestion
    /// <param name="items">The items to increase by number.</param>
```